pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -56,6 +56,10 @@ type TaskRunSpec struct {
 	// Workspaces is a list of WorkspaceBindings from volumes to workspaces.
 	// +optional
 	Workspaces []WorkspaceBinding `json:"workspaces,omitempty"`
+
+	// SchedulerName specifies the scheduler to be used to dispath the Pod
+	// +optional
+	SchedulerName string `json:"schedulerName"`
```

Comment:
I think we should put this in PodTemplate, that's where we've been putting fields that are specific to how the underlying pod will be configured.

Reply:
I have updated the PR accordingly.